google/ggrc-core
# GGRC-5895 Prevent mapping any object to Assessment template

# Issue description

Hide Unified mapper for Assessment Template.

# Steps to test the changes

1. Open any audit page
2. Create Assessment template
3. Hover over gear icon in Assessment template first tier in treeView
4. Click 'Map to this Object': Unified Mapper is displayed 

**Actual Result:** Map to this object' is displayed for Assessment template
**Expected Result:** Hide 'Map to this object' for Assessment template

# Solution description

Remove `mapping-limit` mixin from Assessment template.

# Sanity checklist




# PR Review checklist
